src/Neo/SmartContract/Native/CryptoLib.BLS12_381.cs

```diff
@@ -1,15 +1,36 @@
 using Neo.Cryptography.BLS12_381;
+using Neo.VM.Types;
 using System;
+using System.Runtime.InteropServices;
 using System.Runtime.Versioning;
 
 namespace Neo.SmartContract.Native;
 
 partial class CryptoLib
 {
-    private const int G1 = 48;
-    private const int G2 = 96;
+    private const int G1 = 104;
+    private const int G2 = 200;
     private const int Gt = 576;
 
+    /// <summary>
+    /// The implementation of System.Crypto.GetPoint.
+    /// Convert data to InteropInterface type.
+    /// </summary>
+    /// <param name="g">G point as byteArray</param>
+    /// <returns></returns>
+    [ContractMethod(CpuFee = 1 << 19)]
+    [RequiresPreviewFeatures]
+    public static InteropInterface Bls12381GetPoint(byte[] g)
+    {
+        return g.Length switch
+        {
+            G1 => new InteropInterface(g),
+            G2 => new InteropInterface(g),
+            Gt => new InteropInterface(g),
```

Comment:
Why no just use an integer?

Reply:
Integer？

Reply:
ByteArray sorry

Reply:
I see byte arrays used elsewhere, like LedgerContract. Does it work better with Byte String?

Reply:
We should use `InteropInterface` here because we don't want to expose the inner structures.